demo/debug_panel.tscn

```diff
@@ -16,9 +16,9 @@ border_width_bottom = 1
 border_color = Color(0.160034, 0.223886, 0.359375, 1)
 
 [node name="DebugPanel" type="VBoxContainer"]
-custom_minimum_size = Vector2(0, 400)
+custom_minimum_size = Vector2(0, 250)
 offset_right = 258.0
-offset_bottom = 369.0
+offset_bottom = 400.0
```

Comment:
is this change related to the code cleanup?

Reply:
mistake, sorry, but had top do that in order to display things properly on my screen